docs/api/apiv3/components/schemas/non_working_day_model.yml

```diff
@@ -0,0 +1,40 @@
+# Schema: NonWorkingDayModel
+---
+type: object
+required:
+  - _type
+  - date
+  - name
+properties:
+  _type:
+    type: string
+    enum:
+      - NonWorkingDay
+  date:
+    type: string
+    format: date
+    description: Date of the non-working day.
+  name:
+    type: string
+    description: Descriptive name for the non-working day.
+  _links:
+    type: object
+    required:
+      - self
+    properties:
+      self:
+        allOf:
+          - $ref: './link.yml'
+          - description: |-
+              This non working day. If the non-working day is recurring like a weekend day, the resource will point to a
+              week day.
+
+              **Resource**: NonWorkingDay or WeekDay
```

Comment:
Well, we had this distinction between a recurring non-working day, which is a `WeekDay` with `working == false`, like Sundays, and the non-working days being set manually and ever changing, like the Easter Monday and it is represented by a `NonWorkingDay`. At some point, @Kharonus brought the idea of having only a `Day`, with a `working` attribute, and it would link to either the `NonWorkingDay` or the `WeekDay`. Maybe a `reason` link, or something similar.

But you are right. If it is a `NonWorkingDay` model, and it links to self, then it should link to the same `NonWorkingDay`